module/Entra/Microsoft.Entra/DirectoryManagement/Get-EntraPartnerInformation.ps1

```diff
@@ -19,9 +19,9 @@ function Get-EntraPartnerInformation {
             $params.Keys | ForEach-Object { "$_ : $($params[$_])" } | Write-Debug
             Write-Debug("=========================================================================`n")
             if ([string]::IsNullOrWhiteSpace($TenantId)) {
-                $TenantID = ((invoke-mggraphrequest -Method GET -Uri "https://graph.microsoft.com/v1.0/organization").value).id
+                $TenantID = ((invoke-mggraphrequest -Method GET -Uri "/v1.0/organization").value).id
```

Comment:
Could be a good time to fix casing of `invoke-mggraphrequest` in this and line 24.